distribution/std-lib/Database/src/Connection/Internal/FakeTestConnection.enso

```diff
@@ -0,0 +1,12 @@
+type FakeTestConnection
```

Comment:
are you sure this should even be in this library? not some testing utils?
also `CamelCaseBad`.